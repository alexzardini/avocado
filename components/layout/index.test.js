import React from 'react'
import renderer from 'react-test-renderer'

import Layout from './index.js'

describe('With Snapshot Testing', () => {
  it('Layout struct snapshot', () => {
    const tree = renderer.create(<Layout />).toJSON()
    expect(tree).toMatchSnapshot();
    expect(tree).toMatchSnapshot()
  })
})