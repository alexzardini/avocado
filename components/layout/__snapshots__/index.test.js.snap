// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Layout struct snapshot 1`] = `
<div
  className="jsx-2863305796 root"
>
  <header
    className="jsx-2698932620"
  >
    <h2
      className="jsx-2698932620"
    >
      Carrinho turbo
    </h2>
  </header>
  <div
    className="jsx-3180765612 container"
  />
</div>
`;

exports[`With Snapshot Testing Layout struct snapshot 2`] = `
<div
  className="jsx-2863305796 root"
>
  <header
    className="jsx-2698932620"
  >
    <h2
      className="jsx-2698932620"
    >
      Carrinho turbo
    </h2>
  </header>
  <div
    className="jsx-3180765612 container"
  />
</div>
`;
